flink-connectors/flink-connector-kafka-0.9/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer09.java

```diff
@@ -311,4 +323,19 @@ private static void setDeserializer(Properties props) {
 		props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, deSerName);
 		props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, deSerName);
 	}
+
+
+	// ------------------------------------------------------------------------
+	// Rate Limiter specific methods
+	// ------------------------------------------------------------------------
+
+	/**
+	 * Set a rate limiter to ratelimit bytes read from Kafka.
+	 * @param kafkaRateLimiter
+	 * @param maxBytesPerSecond
+	 */
+	public void setRateLimiter(FlinkConnectorRateLimiter kafkaRateLimiter, long maxBytesPerSecond) {
```

Comment:
Why the second parameter? That should be a setter on `DefaultKafkaRateLimiter`

Reply:
Fixed.